index.md

```diff
@@ -107,7 +107,9 @@ Base Tools
 -	[PHPStorm integration](https://github.com/xwp/wp-phpstorm)
 -	[PHPStorm code style](https://github.com/Automattic/PhpStorm-Resources)
 -	[Brackets](https://github.com/Tusko/brackets-wp-hint/)
--	[Atom](https://atom.io/)
+  -	[Atom](https://atom.io/) - Search 'wordpress' keyword on [Atom packages repository](https://atom.io/packages)
```

Comment:
Let's drop this line - the point here is to link directly do useful WP packages, not link to an IDE and then tell folks how to search.

Reply:
Double that.